examples/04_benchmarking_v2.ipynb

```diff
@@ -0,0 +1,288 @@
+       "      <td>0.123298</td>\n",
+       "      <td>0.185644</td>\n",
+       "      <td>0.184654</td>\n",
+       "      <td>0.118918</td>\n",
+       "      <td>0.051265</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                       validation_id                 model_id  \\\n",
+       "0  [dataset=load_airline]_[cv_splitter=ExpandingW...  NaiveForecaster-last-v1   \n",
+       "1  [dataset=load_airline]_[cv_splitter=ExpandingW...  NaiveForecaster-mean-v1   \n",
+       "\n",
+       "   runtime_secs  MeanSquaredPercentageError_fold_0_test  \\\n",
+       "0      0.121201                                0.024532   \n",
+       "1      0.126832                                0.049681   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_1_test  \\\n",
+       "0                                0.020831   \n",
+       "1                                0.073700   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_2_test  \\\n",
+       "0                                0.001213   \n",
+       "1                                0.053520   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_3_test  \\\n",
+       "0                                0.014950   \n",
+       "1                                0.081063   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_4_test  \\\n",
+       "0                                0.031067   \n",
+       "1                                0.138163   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_5_test  \\\n",
+       "0                                0.008373   \n",
+       "1                                0.145125   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_6_test  \\\n",
+       "0                                0.007972   \n",
+       "1                                0.154337   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_7_test  \\\n",
+       "0                                0.000009   \n",
+       "1                                0.123298   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_8_test  \\\n",
+       "0                                0.028191   \n",
+       "1                                0.185644   \n",
+       "\n",
+       "   MeanSquaredPercentageError_fold_9_test  MeanSquaredPercentageError_mean  \\\n",
+       "0                                0.003906                         0.014104   \n",
+       "1                                0.184654                         0.118918   \n",
+       "\n",
+       "   MeanSquaredPercentageError_std  \n",
+       "0                        0.011451  \n",
+       "1                        0.051265  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results_df = benchmark.run(\"./forecasting_results.csv\")\n",
```

Comment:
In my run, `forecasting_results.csv` shows up in the root directory of the repo. In this by design? If yes, I think it's better in the working directory. 

Though I can easily `pd.read_csv` this file, why are we not saving it to memory?